### PR TITLE
feat(translations): Add a new 'translations' property

### PR DESCRIPTION
**Pull request by jeromecambon**
Add a new 'translations' property to provide the translations to the web components.

Covers [UID-571](https://bonitasoft.atlassian.net/browse/UID-571)

### CODE REVIEW

File: components/uid-input/test/uid-input.test.ts [hunk 1]
```diff
@@ -146,4 +146,29 @@ describe('UidInput', () => {
...
+    expect(label.textContent).equals("Default label");
+  });
+
+  it('Should keep the default label when attribute localization does not contain the correct key', async () => {
+    uidInput = await fixture(html`
+      <uid-input localization='{"DefaultLabel": "Label par défaut"}'></uid-input>
+    `);
+    const label = uidInput.shadowRoot.querySelector('label');
+    expect(label.textContent).equals("Default label");
+  });
```
Comment (julienmege) [inline components/uid-input/test/uid-input.test.ts:171]: Maybe add a test "Should keep the default label when attribute localization contain Empty translation?"

File: components/uid-input/src/UidInput.ts [hunk 2]
```diff
@@ -77,6 +76,10 @@ export class UidInput extends LabeledElement {
       `
     ];
 
+  async attributeChangedCallback(name: string, old: string | null, value: string | null): Promise<void> {
```
Comment (julienmege) [inline components/uid-input/src/UidInput.ts:79]: Is this required to implement this function? as it only call the Super?

Comment (jeromecambon) [reply components/uid-input/src/UidInput.ts:79]: No, this is indeed not needed

File: components/uid-input/test/uid-input.test.ts (hunk 1 above)
Comment (jeromecambon) [reply components/uid-input/test/uid-input.test.ts:171]: So you mean to check if the translation is not empty. Yes, we can add this.

Comment (julienmege) [reply components/uid-input/test/uid-input.test.ts:171]: yes add the test and the implementation, as it work like this in UID today?
